doc/api/deprecations.md

```diff
@@ -95,12 +95,18 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
```

Comment:
@jasnell No problem.  Improvements can come iteratively :)  Do you want me to take a crack at writing that update, or do you want to tackle it?